src/schema/types/Query.ts

```diff
@@ -0,0 +1,12 @@
+import { queryType } from 'nexus'
+import {
+  allUser,
+  user
+} from './queries'
+
+export const Query = queryType({
+  definition(t) {
+    t.list.field('allUser', allUser)
+    t.field('user', user)
+  },
+})
```

Comment:
똑같이 한번에 묶어서 import 했습니다
```suggestion
import * as q from './queries'

export const Query = queryType({
  definition(t) {
    t.list.field('allUser', q.allUser)
    t.field('user', q.user)
  },
})
```